app/services/DocumentType.scala

```diff
@@ -18,7 +18,10 @@ package services
 
 import utils.JsonFile
 
-case class DocumentType(description: String, code: String)
+case class DocumentType(description: String, code: String) {
+
+  def asText: String = description + " - " + code
```

Comment:
`toString`

Reply:
I'll stay with this, everyone has different opinion about it. Something that will be better to discuss